builder/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "git@github.com-personal:Jefiozie/ngx-aws-deploy.git"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "json2ts ng-add/schema.json > ng-add/schema.d.ts && json2ts deploy/schema.json > deploy/schema.d.ts && tsc",
```

Comment:
Maybe its better to split this in a seperate npm script? What dou you think about that?

Reply:
will do.